pkg/services/govmomi/ipam/status_test.go

```diff
@@ -251,7 +251,7 @@ func Test_BuildState(t *testing.T) {
 
 	before := func() {
 		ctx = context.Background()
-		vmCtx = *fake.NewVMContext(fake.NewControllerContext(fake.NewControllerManagerContext()))
+		vmCtx = *fake.NewVMContext(ctx, fake.NewControllerContext(fake.NewControllerManagerContext()))
```

Comment:
```suggestion
		vmCtx = *fake.NewVMContext(ctx, fake.NewControllerContext(fake.NewControllerManagerContext()))
		ctx = context.Background()
```

Reply:
I didn't quite get why we're switching these two lines

Reply:
whoops, my bad :-) you are right. It doesn't make sense.